dune/xt/la/container/vector-interface.hh

```diff
@@ -156,12 +156,7 @@ public:
     return this->as_imp()[ii];
   }
 
-  /**
-   * \brief   The dimension of the vector.
-   * \return  The dimension of the vector.
-   * \see     size()
-   */
-  inline size_t dim() const
+  inline size_t DUNE_DEPRECATED_MSG("Use size() instead (09.04.2018)!") dim() const
```

Comment:
Do you mind using the new macro for deprecations? https://github.com/dune-community/dune-xt-common/issues/13